controllers/util/storageconsumer.go

```diff
@@ -14,7 +14,7 @@ import (
 const (
 	// Reserved RadosNamespaceName for internal use and their representation at different layes
 	ImplicitRbdRadosNamespaceName = "<implicit>"
-	Is419AdjustedAnnotationKey    = "ocs.openshift.io/4_19-adjusted"
+	TicketAnnotation              = "ocs.openshift.io/provider-onboarding-ticket"
```

Comment:
This should be already defined somewhere in ocs-operator as were already using it in 4.18